compiler/src/yul/runtime/functions.rs

```diff
@@ -0,0 +1,161 @@
+use yultsur::*;
```

Comment:
interesting -- is this runtime not defined elsewhere -- like in `yultsur`? 

it may not be but it would be nice to have a single `yul` runtime available to rust programs that we can just import as a crate

Reply:
Not that I'm aware of. It would definitely be useful to have those somewhere tho and `yultsur` seems like the right place.

Any input on this @axic? :)